src/tools/pub/dependencies.md

```diff
@@ -441,21 +441,34 @@ Use [caret syntax](#caret-syntax)
   but where a new version `2.x` would be a major upgrade
   that isn't semantically compatible with `1.x` versions. 
 
-Depend on the latest stable package versions
-: Use [`pub upgrade`][] to update to the latest package versions
+### Depend on the latest stable package versions
+ Use [`pub upgrade`][] to update to the latest package versions
   that your pubspec allows.
   To identify dependencies in your app or package that
   aren't on the latest stable versions,
   use [`pub outdated`][].
 
-Test whenever you update package dependencies
-: If you run [`pub upgrade`][] without updating your pubspec,
+### Test whenever you update package dependencies
+ If you run [`pub upgrade`][] without updating your pubspec,
   the API should stay the same
   and your code should run as before—but test to make sure.
   If you modify the pubspec and update to a new major version,
   then you might encounter breaking changes,
   so you need to test even more thoroughly.
 
+### Verify the integrity of downloaded packages
+ When getting new dependencies, use the [`--enforce-lockfile`][]
+  option to ensure the extracted package content matches
```

Comment:
Is the indent level change intended?

Reply:
Honestly I don't know 😅 

If you look at [the current md](https://github.com/dart-lang/site-www/blob/main/src/tools/pub/dependencies.md?plain=1#L434), there's some kind of colon-indent formatting going on. But if you look at the [rendered page](https://dart.dev/tools/pub/dependencies#best-practices) it doesn't seem to actually format  anything? So I kept the indents just in case but that's probably not necessary.